data/database.py

```diff
@@ -1012,6 +1012,13 @@ class QuotaRepositorySize(BaseModel):
     backfill_complete = BooleanField(null=False, default=False)
 
 
+class QuotaRegistrySize(BaseModel):
+    size_bytes = BigIntegerField(null=False, default=0)
+    running = BooleanField(null=False, default=False)
+    queued = BooleanField(null=False, default=False)
```

Comment:
Why do you need a `queued` state if  there's already a `running` state? i.e what's the difference between use cases between the two.

Reply:
Couldn't you do that with only one state, with either `running` set or not?

Reply:
well if running is false it doesn't mean it should be ran. There needs to be a field saying whether the calculation is currently running or not so no other job picks it up and another field saying that the calculation is queued to be ran. If both are false nothing should happen. So really there's 3 states - do nothing, queued, running. I don't think we can represent that as a single boolean.

Reply:
this isn't a queue worker, it's a regular worker that periodically checks the table. This way when attempting to queue if it's already been queued it's just a single field check whether `queued == true`.